pasta_eln/gui.py

```diff
@@ -49,44 +49,43 @@ def __init__(self) -> None:
     # Menubar
     menu = self.menuBar()
     projectMenu = menu.addMenu("&Project")
-    Action('&Export .eln', self, [Command.EXPORT], projectMenu)
-    Action('&Import .eln', self, [Command.IMPORT], projectMenu)
+    Action('&Export .eln',                   self, [Command.EXPORT],         projectMenu)
```

Comment:
Since it's not a standard kind of formatting, do we have to waste time doing such fancy formatting? :) I personally try to go for the IDE format settings and it would be easier also for the collaborators when we follow the standard python formatting from IDE..

Reply:
It takes me 5 min and from my point of view, it is not a waste as it eases the reading greatly. I also do not think we should follow the IDEs and their limits blindly. IDEs cannot see context and therefore they optimize following some rules. Humans are not IDEs and they read differently.